tools/refguide_check.py

```diff
@@ -102,6 +105,14 @@
     'numpy.lib.Repository',
 ])
 
+# skip non-numpy RST files, historical release notes
+RST_SKIPLIST = [
+    'scipy-sphinx-theme',
+    'sphinxext',
+    'neps',
+    'release',
```

Comment:
Does this exclude upcoming changelog snippets too? They should probably be included.

Reply:
it's complicated :). On the one hand, we want to skip all of `doc/sphinxext`, including subdirectories. On the other hand, we want to skip the files in `doc/release`, but dive into `doc/release/upcoming_changes`. Done in 2c11348